package.json

```diff
@@ -20,7 +20,7 @@
     "make:lib:js": "mkdirp lib && babel src --out-dir lib --ignore=__tests__/* --source-maps",
     "make:lib:css": "mkdirp lib && babel-node scripts/styles.js && SASS_ENV=ie babel-node scripts/styles.js && babel-node scripts/postcss.js && SASS_ENV=ie babel-node scripts/postcss.js",
     "prepublishOnly": "npm run make:lib",
-    "setup": "npm install && npm link && rm -rf node_modules/react && rm -rf node_modules/react-dom && cd examples/simple && npm install && npm link react-plotly.js-editor",
+    "setup": "npm install && cd examples/simple && npm install && cd ../redux && npm install && cd ../async-data && npm install",
```

Comment:
Ah OK if we're doing things this way, then we could consider skipping `setup` and adding these as a post-install hook

Reply:
hmm, you know I was just adding this in so that people could have an easy command to run to get an example running and play around with the code a little. 

So thinking back of the original motivations, I think I'll remove the npm installs to the redux and async-data examples, and keep that npm link to the simple repo.